### ISSUE TITLE
Can't install under Ubuntu 20.04 (Linux Mint)

### ISSUE THREAD

**Issue by All3xJ** [vagnum08/cpupower-gui#68]
**Describe the bug**
I followed "Debian/Ubuntu and derivatives" guide in the README.
```
echo 'deb http://download.opensuse.org/repositories/home:/erigas:/cpupower-gui/xUbuntu_20.04/ /' | sudo tee /etc/apt/sources.list.d/home:erigas:cpupower-gui.list
curl -fsSL https://download.opensuse.org/repositories/home:erigas:cpupower-gui/xUbuntu_20.04/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home:erigas:cpupower-gui.gpg > /dev/null
sudo apt update
```

I receive this error after apt update:
```
W: No Hash entry in the Release /var/lib/apt/lists/partial/download.opensuse.org_repositories_home:_erigas:_cpupower-gui_xUbuntu%5f20.04_InRelease file
E: The "http://download.opensuse.org/repositories/home:/erigas:/cpupower-gui/xUbuntu_20.04 InRelease" repository only provides weak security information.
N: Upgrading from this repository cannot be done securely and is therefore disabled by default.
N: See the apt-secure(8) man page for repository creation and user configuration.
```

**Setup (please complete the following information):**
 - Distro: LinuxMint (Ubuntu 20.04)

**Comment by supdrewin** [comment]
repos `xUbuntu_20.04` is now empty, try this following instead:
``` shell
echo 'deb https://download.opensuse.org/repositories/home:/erigas:/cpupower-gui/xUbuntu_20.10/ /' | sudo tee /etc/apt/sources.list.d/home:erigas:cpupower-gui.list
curl -fsSL https://download.opensuse.org/repositories/home:erigas:cpupower-gui/xUbuntu_20.10/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home:erigas:cpupower-gui.gpg

```

**Comment by All3xJ** [comment]
> Also, you may need to upgrade your ubuntu to 20.10 or later because `focal` don't provide a
> `libhandy-1-dev` package for install the needed depends.

Isn't there any way to install it if I am not under 20.10 without compile it?

**Comment by shawnwall** [comment]
noting that if ubuntu 20.04 is not supported, the README should probably be updated to remove/update the example install lines for "Debian/Ubuntu and derivatives".

**Comment by vagnum08** [comment]
This was resolved in #80.

Debian packages for Linux Mint 20.3 are available from the Releases page.
